### PR TITLE
Fixed lockfiles not being added to the "npm version" commit

### LINKED ISSUE #17707: package-lock modified but not committed to git repo during "npm version [major|minor|patch]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [x] Other (_see below for feature requests_):

#### What's going wrong?

When using "npm version [major | minor | patch]" on a project in a git repo, npm will bump the version then commit the package.json changes to git.  However, package-lock.json also gets updated with the bumped version but is not included in the commit.  This leaves the git repo with unstaged changes after a version bump, requiring a separate commit or amend.

#### How can the CLI team reproduce the problem?

Create a node project inside a git repo.  Install a dependency to cause the creation of a package-lock.json.  Run "npm version patch", and observe the creation of a new git commit, and the presence of package-lock as an unstaged change under "git status".

### supporting information:

 - `npm -v` prints: 5.1.0
 - `node -v` prints: 6.9.1
 - `npm config get registry` prints: (internal corp proxy registry)
 - Windows, OS X/macOS, or Linux?: Windows 10

### PR DESCRIPTION
**Pull request by markpeterfejes**
Hey!

This small PR fixes #17707 by executing the `git add` commands sequentially instead of concurrently, which sometimes causes the aforementioned error.

I'm not really sure that this is the prettiest way to implement this, change suggestions are very welcome!

Thank you!

### CODE REVIEW

File: lib/version.js [hunk 1]
```diff
@@ -282,25 +282,34 @@ function _commit (version, localData, cb) {
...
         'tag', npm.config.get('tag-version-prefix') + version,
-        flag, message
+        flagForTag, message
       ], options)
     }
   }).nodeify(cb)
 }
 
+function stagePackageFiles (localData, options) {
+  return new BB(function (resolve, reject) {
```
Comment (zkat) [inline lib/version.js]: You don't need a promise constructor here. Just:

```
return addLocalFile('package.json', options, false).then(() => {
  if (localData.hasShrinkwrap) {
    return addLocalFile('npm-shrinkwrap.json', options, false)
  } else if (localData.hasPackageLock) {
    return addLocalFile('package-lock.json', options, false)
  }
})
```

**Review by zkat — CHANGES_REQUESTED** [review]
needs a small mostly-stylistic change

File: lib/version.js (hunk 1 above)
Comment (markpeterfejes) [reply lib/version.js]: I'm doing this right now! Is there a possibility that this could be merged to the 5.3.0 relese today?

Comment (zkat) [reply lib/version.js]: probably! I'll take a look in the AM and see if this patch is ready to merge. 👍

Comment (markpeterfejes) [reply lib/version.js]: Thank you very much!

**Comment by markpeterfejes** [comment]
Should be ready for release :)

**Review by zkat — APPROVED** [review]
changes look good, everything seems to work. gj, and thanks for the contrib! 🎉

**Pull request merged**